.travis.yml

```
@@ -56,7 +36,7 @@ matrix:
 
     - os: osx
       name: Danger with SPM
-      osx_image: xcode10
+      osx_image: xcode11
```

what do you think about using 11.2.1? 11.0 is pretty outdated, but 11.2.1 has Swift 5.1.2 though
```suggestion
      osx_image: xcode11.2
```

---

Yes! Thank you very much :)